Rider/app/src/main/java/com/example/rider/MainActivity.kt

```diff
@@ -11,6 +12,41 @@ class MainActivity : AppCompatActivity() {
     lateinit var mainFragment: MainFragment
     lateinit var secondFragment: SecondFragment
     lateinit var thirdFragment: ThirdFragment
+=======
+import android.view.MotionEvent
+import android.view.View
+import android.widget.Button
+import android.widget.LinearLayout
+import android.widget.TextView
+import android.widget.Toast
+import androidx.fragment.app.FragmentTransaction
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import kotlinx.android.synthetic.*
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.upcoming_ride.*
+
+const val EXTRA_MESSAGE = "com.example.rider.MESSAGE"
+
+class MainActivity : AppCompatActivity() {
+
+    fun TextView.onRightDrawbaleClicked(onClicked: (view: TextView) -> Unit) {
+        this.setOnTouchListener { v, event ->
+            var hasConsumed = false
+            if (v is TextView) {
```

Comment:
Not sure why this check is here?
`v` is not the best name.
Also checking the class of a variable is bad practice.